### ISSUE TITLE
Reading mime contents of .msg file type attachments

### ISSUE THREAD

**Issue by kamalvk18** [ecederstrand/exchangelib#1255]
Hi, Thanks for the support.

I'm trying to load the attachments from .msg files into the azure blob storage. Whenever there are attachments of excel or pdf types I'm able to get the data from the attachments as bytes and I'm able to upload the file to blob storage.

 But Whenever there is a .msg type attachment. I'm not able to get the mime_content as bytes, I'm getting a Message object instead. Is there any way I can get the mime_content as bytes?

Any help on this will be much appreciated!

Thank you.

**Comment by ecederstrand** [comment]
It's a bit hard to follow what you're trying to do. Can you please post the code you use to read the .msg files?
